tokio/src/runtime/tests/loom_multi_thread_alt.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(tokio_unstable)]
```

Comment:
Maybe we want to put this annotation on files under `tokio/src/runtime/tests/loom_multi_thread_alt/*` as well?

Reply:
isn't `loom_multi_thread_alt.rs` basically the `mod.rs` file for that directory? So I believe by putting the annotation in `loom_multi_thread_alt.rs`, the whole module is affected.

Reply:
Ah my bad, you're right. Annotating against `loom_multi_thread_alt.rs` is enough.